Userland/Libraries/LibVideo/VP9/Enums.h

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright (c) 2021, Hunter Salyer <thefalsehonesty@gmail.com>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+namespace Video::VP9 {
+
+enum FrameType {
+    KEY_FRAME,
+    NON_KEY_FRAME
+};
+
+enum ColorSpace : u8 {
+    CS_UNKNOWN = 0,
+    CS_BT_601 = 1,
+    CS_BT_709 = 2,
+    CS_SMPTE_170 = 3,
+    CS_SMPTE_240 = 4,
+    CS_BT_2020 = 5,
+    CS_RESERVED = 6,
+    CS_RGB = 7
+};
+
+enum ColorRange {
+    STUDIO_SWING,
+    FULL_SWING
+};
+
+enum InterpolationFilter {
+    EIGHTTAP = 0,
+    EIGHTTAP_SMOOTH = 1,
+    EIGHTTAP_SHARP = 2,
+    BILINEAR = 3,
+    SWITCHABLE = 4
+};
+
+enum ReferenceFrame {
+    INTRA_FRAME = 0,
+    LAST_FRAME = 1,
+    GOLDEN_FRAME = 2,
+    ALTREF_FRAME = 3,
```

Comment:
WHY ARE THESE SCREAMING AT ME
IT'S REALLY UNCOMFORTABLE :P

Reply:
@alimpfard These identifiers are written in caps in the VP9 spec.

Reply:
Does that mean we have to keep them like that?
a lot of specs have their own naming conventions, I don't think code needs to conform to them

Reply:
We don't have to, I honestly just like having constants & enum values in all caps. If that's not preferred, I'll totally change them.

Reply:
Technically there's a rule about it in the coding style:
> Enum members should use InterCaps with an initial capital letter.